scripts/convert_bed.py

```diff
@@ -0,0 +1,36 @@
+import pybedtools
```

Comment:
pylint requires to add docstrings on top of every non-empty python file, which I agree is a good practice. It's not clear from the script name what kind of conversion it does. I'd comment every snakemake files as well.